.github/workflows/ci.yaml

```diff
@@ -14,22 +14,28 @@ jobs:
   build:
     env:
       COVER: true
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     steps:
 
     - name: Check out code
       uses: actions/checkout@v4
 
-    - name: Set up Go 1.21
+    - name: Set up Go
       uses: actions/setup-go@v5
       with:
-        go-version: 1.21
+        # renovate: datasource=golang-version depName=golang
+        go-version: 1.21.6
       id: go
 
     - name: Get dependencies
+      env:
+        # renovate: datasource=github-tags depName=golangci/golangci-lint
+        GOLANGCI_LINT_VERSION: v1.55.2
+        # renovate: datasource=github-tags depName=codeclimate/test-reporter
+        CODECLIMATE_VERSION: v0.11.1
       run: |
-        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.55.2
-        curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin ${GOLANGCI_LINT_VERSION}
+        curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-${CODECLIMATE_VERSION}-linux-amd64 > ./cc-test-reporter
```

Comment:
Why would we want to have a fixed version for the code climate api as it is a SaaS we should just use the latest version to be able to upload the reports.